internal/plugin/resource_vault_start_test.go

```diff
@@ -68,6 +68,48 @@ func TestVaultStartConfigOptionalAttrs(t *testing.T) {
 		"usage_gauge_period":        "30m",
 		"maximum_gague_cardinality": 100,
 	})
+	vaultCfg.UI.Set(true)
```

Comment:
I like that we're testing optional attributes being set but most of the time our issues with optional attributes arise when we _think_ they're optional but they really are not. Perhaps we should have another test case that only provides required inputs to ensure that it plans correctly.